server/index.js

```diff
@@ -14,8 +14,8 @@ app.post('/recipe', (req, res) => {
 })
 
 
-app.get('/recipe', (req, res) => {
-  return Comments.getRecipe()
+app.get('/recipe/:id', (req, res) => {
```

Comment:
the route should be /recipe/:id/reviews or something similar; there are many different modules for any given recipe and the route for the API call should reflect the information you're intending to get.